src/Compilation/Config.zig

```diff
@@ -119,6 +119,7 @@ pub const ResolveError = error{
     ZigLacksTargetSupport,
     EmittingBinaryRequiresLlvmLibrary,
     LldIncompatibleObjectFormat,
+    LldIncompatibleWithStage2,
```

Comment:
Why `Stage2`? What does it even mean? What if we described exactly what is happening, namely `LldCannotIncrementallyLink` or something like that? I mean, this is effectively what happens here, isn't it?

Reply:
That's a perfect name thanks!